hack/go.sh

```diff
@@ -55,19 +55,23 @@ build(){
     cd "${REPO_ROOT}"
     for p in ${PKGS}; do
       echo "go build'ing package $p"
-      go build -a "${p}"
+      go build -a "${p}" || RET=$?
     done
+    return $RET
   )
+  return $?
```

Comment:
It looks like the goal here is to try to build every pkg, even when 1 or more fails. But the err returned is for the last failure only. Perhaps we should fail on the 1st build error, or return a list of error strings of a format like: "$p: $?". What do you think?

Reply:
My intention is to report the build status to CI.
If use fail-fast manner, it is sufficient for the purpose, however, it breaks current behavior which tries to build all packages.
I don't want to change the behavior because some tools may depend on such behavior.
And I personally prefer that all build error is reported.

For 2nd suggestion, it is not effective, IMO.
The script already does echo for package names, and Go also reports which package fails.